multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -89,13 +89,13 @@ def get_basic_automm_config():
     }
 
 
-def get_automm_preset(preset: str):
+def get_automm_presets(presets: str):
```

Comment:
Do we really need to use the plural form of preset everywhere? I think it's better to keep the singular form in most places except for the one in `.fit()`. No need to propagate the wrong form to other places.